diff --git a/src/ethosu_config_u55.h b/src/ethosu_config_u55.h
index 9330bb1..c1ce4a1 100644
--- a/src/ethosu_config_u55.h
+++ b/src/ethosu_config_u55.h
@@ -22,11 +22,11 @@
 /* Set default values if not manually overriden */
 
 #ifndef NPU_QCONFIG
-#define NPU_QCONFIG 2
+#define NPU_QCONFIG 0
 #endif
 
 #ifndef NPU_REGIONCFG_0
-#define NPU_REGIONCFG_0 3
+#define NPU_REGIONCFG_0 0
 #endif
 
 #ifndef NPU_REGIONCFG_1
@@ -58,11 +58,11 @@
 #endif
 
 #ifndef AXI_LIMIT0_MAX_BEATS_BYTES
-#define AXI_LIMIT0_MAX_BEATS_BYTES 0x0
+#define AXI_LIMIT0_MAX_BEATS_BYTES 0x1
 #endif
 
 #ifndef AXI_LIMIT0_MEM_TYPE
-#define AXI_LIMIT0_MEM_TYPE 0x0
+#define AXI_LIMIT0_MEM_TYPE 0x2
 #endif
 
 #ifndef AXI_LIMIT0_MAX_OUTSTANDING_READS
@@ -74,11 +74,11 @@
 #endif
 
 #ifndef AXI_LIMIT1_MAX_BEATS_BYTES
-#define AXI_LIMIT1_MAX_BEATS_BYTES 0x0
+#define AXI_LIMIT1_MAX_BEATS_BYTES 0x1
 #endif
 
 #ifndef AXI_LIMIT1_MEM_TYPE
-#define AXI_LIMIT1_MEM_TYPE 0x0
+#define AXI_LIMIT1_MEM_TYPE 0x2
 #endif
 
 #ifndef AXI_LIMIT1_MAX_OUTSTANDING_READS
@@ -90,11 +90,11 @@
 #endif
 
 #ifndef AXI_LIMIT2_MAX_BEATS_BYTES
-#define AXI_LIMIT2_MAX_BEATS_BYTES 0x0
+#define AXI_LIMIT2_MAX_BEATS_BYTES 0x1
 #endif
 
 #ifndef AXI_LIMIT2_MEM_TYPE
-#define AXI_LIMIT2_MEM_TYPE 0x0
+#define AXI_LIMIT2_MEM_TYPE 0x2
 #endif
 
 #ifndef AXI_LIMIT2_MAX_OUTSTANDING_READS
@@ -106,11 +106,11 @@
 #endif
 
 #ifndef AXI_LIMIT3_MAX_BEATS_BYTES
-#define AXI_LIMIT3_MAX_BEATS_BYTES 0x0
+#define AXI_LIMIT3_MAX_BEATS_BYTES 0x1
 #endif
 
 #ifndef AXI_LIMIT3_MEM_TYPE
-#define AXI_LIMIT3_MEM_TYPE 0x0
+#define AXI_LIMIT3_MEM_TYPE 0x2
 #endif
 
 #ifndef AXI_LIMIT3_MAX_OUTSTANDING_READS
